tool/teleport/common/teleport.go

```diff
@@ -278,8 +280,13 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dbConfigureCreate.Flag("uri", "Address the proxied database is reachable at.").StringVar(&dbConfigCreateFlags.StaticDatabaseURI)
 	dbConfigureCreate.Flag("labels", "Comma-separated list of labels for the database, for example env=dev,dept=it").StringVar(&dbConfigCreateFlags.StaticDatabaseRawLabels)
 	dbConfigureCreate.Flag("aws-region", "(Only for AWS-hosted databases) AWS region RDS, Aurora, Redshift, Redshift Serverless, ElastiCache, or MemoryDB database instance is running in.").StringVar(&dbConfigCreateFlags.DatabaseAWSRegion)
+	dbConfigureCreate.Flag("aws-account-id", "(Only for Keyspaces or DynamoDB) AWS Account ID.").StringVar(&dbConfigCreateFlags.DatabaseAWSAccountID)
 	dbConfigureCreate.Flag("aws-external-id", "(Only for AWS-hosted databases) Optional AWS external ID to use when assuming AWS roles.").StringVar(&dbConfigCreateFlags.DatabaseAWSExternalID)
 	dbConfigureCreate.Flag("aws-redshift-cluster-id", "(Only for Redshift) Redshift database cluster identifier.").StringVar(&dbConfigCreateFlags.DatabaseAWSRedshiftClusterID)
+	dbConfigureCreate.Flag("aws-rds-cluster-id", "(Only for RDS Aurora) RDS Aurora database cluster identifier.").StringVar(&dbConfigCreateFlags.DatabaseAWSRDSClusterID)
+	dbConfigureCreate.Flag("aws-rds-instance-id", "(Only for RDS) RDS database instance identifier.").StringVar(&dbConfigCreateFlags.DatabaseAWSRDSInstanceID)
+	dbConfigureCreate.Flag("aws-elasticache-group-id", "(Only for ElastiCache) ElastiCache replication group identifier.").StringVar(&dbConfigCreateFlags.DatabaseAWSElastiCacheGroupID)
+	dbConfigureCreate.Flag("aws-memorydb-cluster-name", "(Only for MemoryDB) MemoryDB cluster name.").StringVar(&dbConfigCreateFlags.DatabaseAWSMemoryDBClusterName)
```

Comment:
I'd prefer keeping them for consistency and to prevent a scenario where a configuration value can't be specified but it's required to pass the config verification checks.

Re: `(Optional for XXX)` vs `(Required for YYY)` - I think those cases will naturally become obvious when running the command. It may be that a flag is sometimes required, but depends on the settings for other flags. For instance, `--uri` is technically optional for DynamoDB, but only if `--aws-region` is specified. So determining optional vs required upfront in the help string isn't really possible to do, at least not accurately for all cases.

I definitely missed adding a lot of values from `lib/config/fileconf.Database` though, and I'd prefer we have them all...

Reply:
> So determining optional vs required upfront in the help string isn't really possible to do, at least not accurately for all cases.

Agreed. Though when I read through `teleport db configure create -h`, it feels to me that one of those IDs is required. Ideally, we want to save the time that spend on grabbing these ids. Could be me just me though.

Reply:
Maybe we should indicate where the flag applies to static db config only? After all with these AWS databases, we prefer users to setup discovery right?

Reply:
Ya, discovery should be preferred. Though the "ids" are optional even for static dbs. Only need the right AWS `--uri`. The "ids" are usually redundant. 

Some Azure static dbs do require certain fields though, sigh.